addon/MetadataStore.js

```diff
@@ -525,6 +525,38 @@ MetadataStore.prototype = {
   }),
 
   /**
+   * Find the oldest entry in the database
+   *
+   * Returns the timestamp of the oldest entry in the database
+   */
+  asyncGetOldestInsert: Task.async(function*() {
+    let timestamp = null;
+    try {
+      const entry = yield this.asyncExecuteQuery("SELECT created_at FROM page_metadata ORDER BY created_at LIMIT 1");
```

Comment:
Although they are equivalent,  let's use this more obvious one `SELECT min(created_at) FROM page_metadata`